content/help/ops/setup/injection/index.md

```diff
@@ -43,132 +43,132 @@ of injected sidecar when it was.
 1. Check the webhook's `namespaceSelector` to determine whether the
    webhook is scoped to opt-in or opt-out for the target namespace.
 
-The `namespaceSelector` for opt-in will look like the following:
-
-{{< text bash >}}
-$ kubectl get mutatingwebhookconfiguration istio-sidecar-injector -o yaml | grep "namespaceSelector:" -A5
-  namespaceSelector:
-    matchLabels:
-      istio-injection: enabled
-  rules:
-  - apiGroups:
-    - ""
-{{< /text >}}
-
-The injection webhook will be invoked for pods created
-in namespaces with the `istio-injection=enabled` label.
-
-{{< text bash >}}
-$ kubectl get namespace -L istio-injection
-NAME           STATUS    AGE       ISTIO-INJECTION
-default        Active    18d       enabled
-istio-system   Active    3d
-kube-public    Active    18d
-kube-system    Active    18d
-{{< /text >}}
-
-The `namespaceSelector` for opt-out will look like the following:
-
-{{< text bash >}}
-$ kubectl get mutatingwebhookconfiguration istio-sidecar-injector -o yaml | grep "namespaceSelector:" -A5
-  namespaceSelector:
-    matchExpressions:
-    - key: istio-injection
-      operator: NotIn
-      values:
-      - disabled
-  rules:
-  - apiGroups:
-    - ""
-{{< /text >}}
-
-The injection webhook will be invoked for pods created in namespaces
-without the `istio-injection=disabled` label.
-
-{{< text bash >}}
-$ kubectl get namespace -L istio-injection
-NAME           STATUS    AGE       ISTIO-INJECTION
-default        Active    18d
-istio-system   Active    3d        disabled
-kube-public    Active    18d       disabled
-kube-system    Active    18d       disabled
-{{< /text >}}
-
-Verify the application pod's namespace is labeled properly and (re) label accordingly, e.g.
-
-{{< text bash >}}
-$ kubectl label namespace istio-system istio-injection=disabled --overwrite
-{{< /text >}}
-
-(repeat for all namespaces in which the injection webhook should be invoked for new pods)
-
-{{< text bash >}}
-$ kubectl label namespace default istio-injection=enabled --overwrite
-{{< /text >}}
+    The `namespaceSelector` for opt-in will look like the following:
+
+    {{< text bash >}}
+    $ kubectl get mutatingwebhookconfiguration istio-sidecar-injector -o yaml | grep "namespaceSelector:" -A5
+      namespaceSelector:
```

Comment:
`namespaceSelector` is indented because it is a subfield in the output YAML (i.e. `webhook.clientConfig[0].namespaceSelector`). Below is the actual output of the command when executed in a terminal. 

```bash
$ kubectl get mutatingwebhookconfiguration istio-sidecar-injector -o yaml | grep "namespaceSelector:" -A5
  namespaceSelector:
    matchLabels:
      istio-injection: enabled
  rules:
  - apiGroups:
    - ""
```